pandas/tests/extension/base/setitem.py

```diff
@@ -210,7 +210,8 @@ def test_setitem_integer_array(self, data, idx, box_in_series):
                 [0, 1, 2, pd.NA], True, marks=pytest.mark.xfail(reason="GH-31948")
             ),
             (pd.array([0, 1, 2, pd.NA], dtype="Int64"), False),
-            (pd.array([0, 1, 2, pd.NA], dtype="Int64"), False),
+            # TODO: change False to True?
+            (pd.array([0, 1, 2, pd.NA], dtype="Int64"), False),  # noqa: PT014
```

Comment:
Hm, this is a dupe right?

Reply:
Yup but changing it to `True` fails a few tests so adding a noqa for now

Reply:
Can you open an issue as a followup?

(the noqa makes it look like ruff had a false positive)

Reply:
Sure, https://github.com/pandas-dev/pandas/issues/56727